articles/sentinel/process-events-normalization-schema.md

```diff
@@ -158,18 +159,18 @@ The process event schema references the following entities, which are central to
 | **ActorUserIdType**| Recommended  | String     |  The type of the ID stored in the [ActorUserId](#actoruserid) field. For more information, see [The User entity](normalization.md#the-user-entity). <br><br>Example: `SID`         |
 | **ActorSessionId** | Optional     | String     |   The unique ID of the login session of the Actor.  <br><br>Example: `999`<br><br>**Note**: The type is defined as *string* to support varying systems, but on Windows this value must be numeric. <br><br>If you are using a Windows machine and used a different type, make sure to convert the values. For example, if you used a hexadecimal value, convert it to a decimal value.   |
 | **ActingProcessCommandLine**       | Optional     | String     |   The command line used to run the acting process. <br><br>Example: `"choco.exe" -v`    |
-| **ActingProcessName**              | Optional     | string     |   The file name of the acting process image file. This is usually considered the process name.  <br><br>Example: `C:\Windows\explorer.exe`  |
-| **ActingProcessCompany**       | Optional     | String     |           The company that created the acting process image file.  <br><br> Example: `Microsoft`    |
+| **ActingProcessName**              | Optional     | string     |   The name of the acting process, commonly derived from the image file or executable file used to define the initial code and data mapped into the process' virtual address space.<br><br>Example: `C:\Windows\explorer.exe`  |
```

Comment:
Clarifying to:
`The name of the acting process. This name is commonly derived from the image or executable file that's used to define the initial code and data that's mapped into the process' virtual address space.`